libs/island-ui/core/src/lib/Header/Header.tsx

```diff
@@ -3,26 +3,51 @@ import React, { ReactElement, ReactNode } from 'react'
 import { Logo } from '../Logo/Logo'
 import { Box } from '../Box/Box'
 import { Text } from '../Text/Text'
-import { Button } from '../Button/Button'
 import { Hidden } from '../Hidden/Hidden'
 import { UserMenu } from './UserMenu/UserMenu'
+import { Inline } from '../Inline/Inline'
 import * as styles from './Header.treat'
 
 export interface HeaderProps {
-  authenticated?: boolean
-  language?: string
+  info?: {
+    title: string
+    description?: string
+  }
   logoRender?: (ReactElement: ReactElement) => ReactElement
   logoutText?: string
+  headerItems?: ReactNode
```

Comment:
Making the Header component simpler, with a `headerItems` prop that can be passed in and and renders on the right side.

Reply:
The other props are still here for compatibility with older apps.